BIDebugEngine/BIDebugEngine/BIDebugEngine.vcxproj

```diff
@@ -388,6 +413,73 @@
     <ClCompile Include="..\..\intercept\src\client\intercept\shared\client_types.cpp" />
     <ClCompile Include="..\..\intercept\src\client\intercept\shared\containers.cpp" />
     <ClCompile Include="..\..\intercept\src\client\intercept\shared\types.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\arraydata.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_apply.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_catch.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_configclasses.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_configproperties.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_count.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_exitwith.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_findif.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_foreach.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_for_step.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_isnil.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_select.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_sqftry.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_switch.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_waituntil.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\callstack_while.cpp" />
+    <ClCompile Include="..\..\lib\sqfvm\src\codedata.cpp" />
```

Comment:
We don't need any of this. We just want the preprocessor, we don't need the compiler nor the assembly nor the interpreter nor the datatypes